website/docs/r/tm_region.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_tm_region"
+sidebar_current: "docs-vcd-resource-tm-region"
+description: |-
+  Provides a resource to manage Regions in VMware Cloud Foundation Tenant Manager.
+---
+
+# vcd\_tm\_region
+
+Provides a resource to manage Regions in VMware Cloud Foundation Tenant Manager.
+
+~> Only `System Administrator` can create this resource.
+
+## Example Usage
+
+```hcl
+data "vcd_tm_supervisor" "one" {
+  name = "first-supervisor"
+
+  depends_on = [vcd_vcenter.one]
+}
+
+resource "vcd_tm_region" "one" {
+  name                 = "region-one"
+  is_enabled           = true
```

Comment:
I can't seem to disable the region. It still shows Status = READY after successful apply.

```
resource "vcd_tm_region" "test" {
  name                 = "region-lino-tf"
  description          = "Region made by Terraform"
  is_enabled           = false
  nsx_manager_id       = vcd_nsxt_manager.test.id
  supervisor_ids       = [data.vcd_tm_supervisor.test.id]
  storage_policy_names = [var.storage_policy_name]
}
```

Reply:
You cannot, the field doesn't work yet https://github.com/vmware/terraform-provider-vcd/pull/1348/files#diff-b102b402cd712cd79b740b0e8ee9394aab65d13e0c703e4500ce5217cfea1619R195